src/query/compiler.js

```diff
@@ -594,7 +594,7 @@ assign(QueryCompiler.prototype, {
   },
 
   onUsing(clause) {
-    return this.formatter.wrap(clause.column);
+    return '(' + this.formatter.columnize(clause.column) + ')';
```

Comment:
I really don't remember nor how formatter.wrap or formatter.columnize behave, but this seems legit 👍